Automattic/wp-calypso
# Reader: migrate full post content styles to `reader-full-post` block

The new Reader full post is nearly ready to release to all users, but it still relies on a bunch of content styles in the old full post (`reader/full-post/_style.css`).

This PR migrates the styles from the old stylesheet and stops including the old full post stylesheet in the SASS build.

⚠️ This should not be merged until the new full post has launched for all users.

### To test

Load a selection of posts in Reader full post view (e.g. https://wpcalypso.wordpress.com/read/feeds/19198732/posts/1191108767) and ensure there are no regressions with the styling of content.
